browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -12193,7 +13259,13 @@ index 0000000000000000000000000000000000000000..5d19607e019489b33318be50ccdc94c2
 +        eventClickCount,
 +        eventModifiers,
 +        timestamp);
++#if PLATFORM(WPE)
++    m_page.setInterceptDrags(true);
```

Comment:
Can you add a comment why we need this for WPE?

Reply:
done.